src/App.tsx

```diff
@@ -13,6 +15,8 @@ const App = () => (
 			<Route path="/" element={<Home />} />
 			<Route path="/entrees" element={<Entrees />} />
 			<Route path="/appetizers" element={<Appetizers />} />
+			<Route path="/sign-up" element={<SignUp />} />
```

Comment:
Minor but is there any reason there is a space between the word in the curly brackets and the /. symbol?

Reply:
The reason i am using / symbol so that when the user change its location (wants to go to another place in the site) It will show up in the url

Reply:
Thank makes sense, thanks